### DISCUSSION TITLE
How to use my TipTap component within a form element

### DISCUSSION THREAD

**Discussion by darrylferdinands** [ueberdosis/tiptap#810] [General Questions] [answered]
Hey. I wanted to know how to use my tiptap component to work within an el-form-item. I currently have an el-input which has a v-model="description" and want to either pass that as a whole to my content prop or replace that el-input altogether with just my tiptap editor. What is the way to go around this? Currently, my description is not binding to my tiptap even when I pass it as a prop to my content.

**Comment by hanspagel** [comment, marked as answer]
If you want to keep using v-model, find a working solution here: https://github.com/ueberdosis/tiptap/issues/133#issuecomment-482787535

Hope that helps!

**Reply by darrylferdinands** [reply]
Thanks! That worked. I also have another doubt related to the next step which is viewing and setting the content. Currently my code is "start of p tag" class="margin-0" v-for="line in description"
{{ line }}
"end of p tag"
I want to setContent of the tiptap editor to the 'line' variable being passed from the v-for map. How should I go about doing that? Any tips will be appreciated
